Linked several apps to drawable, added 10 new icons, and modified some icons

ZhmieXD opened the pull request:
Okay, so I tried to reopen the previous PR #593 anyway but since this is a re-fork the changes didn't update into it so I created a new one. The previous one was kind of messed up 😅

The changes I did in this PR are basically the same as the previous one, plus 3 new Icons.

# Added and Linked Icons

1. Antutu 3DBench (`com.antutu.benchmark.full`) -> Linked to `@drawable/antutu`
2. AOSPStudio's Files Shortcut (`org.aospstudio.files`) -> Linked to `@drawable/files`
3. TIX ID (`id.tix.android`) -> New, with suggestions by @patrykmichalik from the previous PR applied
4. Liputan6 (`com.woi.liputan6.android`) -> New
5. My Pertamina (`com.dafturn.mypertamina`) -> New
6. Quora (`com.quora.android`) -> New

Preview for the new icons:

![image](https://user-images.githubusercontent.com/32131554/179402942-06622244-e256-4975-a3ed-23ab2a85df4e.png)

In addition to these, to avoid redundancy/duplicating of the icons when viewed on the Lawnicons app itself, in the `grayscale_icon_map.xml` I have:
- Renamed `se.burgerking.ordersapp` (Burger King Sweden) to simply "Burger King"
- Renamed `com.android.filemanager` (File Manager) to simply "Files"

Anyways, thanks beforehand! 😁

patrykmichalik reviewed (changes requested):
Thanks for the PR! The icons look great to me. My only suggestion would be to tweak the paths such that they are aligned to the pixel grid. (This is what I did for the SVG included in my review of PR #593.) 

![image](https://user-images.githubusercontent.com/56888459/179416101-1a2d7f84-0f16-4bed-b911-c90b9d8bdade.png)
![image](https://user-images.githubusercontent.com/56888459/179416107-054f07da-904b-497a-85e1-45bf8ffa8667.png)

ZhmieXD commented:
> Thanks for the PR! The icons look great to me. My only suggestion would be to tweak the paths such that they are aligned to the pixel grid. (This is what I did for the SVG included in my review of PR #593.)
> 
> ![image](https://user-images.githubusercontent.com/56888459/179416101-1a2d7f84-0f16-4bed-b911-c90b9d8bdade.png) ![image](https://user-images.githubusercontent.com/56888459/179416107-054f07da-904b-497a-85e1-45bf8ffa8667.png)

Which icon?

patrykmichalik commented:
The icon in the image is `mypertamina.svg`. However, my suggestion applies to all icons (except `tix_id.svg`, which I’ve already reviewed). I suggest you turn on the pixel grid in your vector editor.

Additionally, I regret to let you know that since I’ll no longer be contributing to Lawnchair ([more info](https://t.me/lawnchairci/1557)), I won’t be able to attend to this pull request anymore. I hope another team member will handle it. Thank you for your interest in contributing!

ZhmieXD commented:
> The icon in the image is `mypertamina.svg`. However, my suggestion applies to all icons (except `tix_id.svg`, which I’ve already reviewed). I suggest you turn on the pixel grid in your vector editor.
> 
> Additionally, I regret to let you know that since I’ll no longer be contributing to Lawnchair ([more info](https://t.me/lawnchairci/1557)), I won’t be able to attend to this pull request anymore. I hope another team member will handle it. Thank you for your interest in contributing!

Well I sort of did apply these suggestions of aligning these icons to the grid but then they became uncentered to the page a little bit, that's the problem ^^;

But okay I'll submit them anyway, and yeah I noticed that you've removed your name from the list of contributors, let's see how other members will handle this. Thanks for all the reviews! 😁

ZhmieXD commented:
Okay, Just applied the revisions for the first 3 icons, and added new icon for Camscanner (`com.intsig.camscanner`). Planning to add more icons though since there are still some of at least the apps that are on my phone and are unthemed 😂😂

Here's the preview for CamScanner :

![image](https://user-images.githubusercontent.com/32131554/187283412-d1ceb4d7-7f9a-44e5-9c4f-ddf3b82be513.png)

Chefski commented on app/src/main/res/xml/grayscale_icon_map.xml:
```diff
@@ -408,7 +413,6 @@
     <icon drawable="@drawable/neo_store" package="com.machiav3lli.fdroid" name="Neo Store" />
     <icon drawable="@drawable/netflix" package="com.netflix.mediaclient" name="Netflix" />
     <icon drawable="@drawable/netzclub" package="com.telefonica.netzclub.csc" name="Netzclub" />
-    <icon drawable="@drawable/newpipe" package="org.schabi.newpipe" name="NewPipe" />
```
Any reason for removing `org.schabi.newpipe`?

ZhmieXD replied:
I think it was a duplicate, and I don't think that's the part of the latest commit though, it could be the ones before that

Chefski replied:
Its still part of the PR. Do you mind re-adding it?

ZhmieXD replied:
Yeah I don't remember deleting it. It's back on the latest commit though, keeping that in.

ZhmieXD replied:
> Its still part of the PR. Do you mind re-adding it?

And by latest commit I meant the ones in this PR in my end, not the last PR.

Chefski reviewed (changes requested):
`quora.svg` seems to be smaller than the 160x160 content area. Mind fixing that? Other than that, icons look great!

![image](https://user-images.githubusercontent.com/100310118/190869820-5ef7c265-2247-4a72-b898-9d460c35b20c.png)

ZhmieXD commented:
Alright, finally got all my installed apps themed now! XD

Yeah sorry I didn't respond to the change request sooner because I wanted to do more icons at once, and that's what I did. Here's the rundown:

- Modified `quora.svg` per @Chefski's request
- Sorted lines alphabetically for Domino's Pizza apps and added more regional versions :

1. Azerbaijan (`com.dominospizza.az`)
2. Germany (`nl.speakap.dominosdeutschland`)
3. Indonesia (`com.phonegap.dominos`)
4. Israel (`il.co.dominos.android`)
5. Middle East (`com.dominos.sa`)

- Added new Icons :

1. Al-Quran Indonesia (`com.andi.alquran.id`) [1]
2. CAF FM Radio (`com.caf.fmradio`) [2]
3. Hijri Calendar (`com.kodelokus.hijri`)
4. Magic Poser (`com.magicposernew`)

### Notes

[1] Sorry I used some SVG for the Quran Kareem calligraphy out of Wikimedia, let me know if you'd like a different shape like for example a simplified front cover of the Quran itself, but for now, I'm sticking with this.
[2] For other contributors, feel free to ink `@drawable/fm_radio` to your FM Radio app, whether it is the specific device/ROM stock one or a 3rd party one.

##  

Thanks once again! 😁

Chefski reviewed (changes requested):
All these icons look great! Unfortunately `quran.svg` will need changes. Other than that, all the icons are fine and don't need any changes :)

ZhmieXD commented:
> All these icons look great! Unfortunately `quran.svg` will need changes. Other than that, all the icons are fine and don't need any changes :)

Well I just saw the latest action where Physics Wallah, a new one added, doesn't really use the project's main icon theme and yet it got accepted right away ^^;

But yeah, modifying `quran.svg` right now. Thought of adding one more Icons as well anyway for a popular Pad Icons and wallpapers pack on Play Store.

ZhmieXD commented:
Alright here's another change I've made:

- Modified `quran.svg` per latest request
- Modified `physics_wallah.svg` from PR #673 to match current style
- Modified `yandex.svg` to have 12px outline width
- Added Oxigen HD (`com.cris87.oxigen`)

Hopefully that's everything. Thanks once again! 😁

Chefski reviewed (approved):
Every icon looks amazing! 🤩 Thanks for contributing

Chefski commented:
Also good work on improving physics wallah

ZhmieXD commented:
> Also good work on improving physics wallah

Umm, excuse me, when will this be merged?

Chefski commented:
I don't have merge perms. I'll message someone to merge now
